### PR TITLE
Update index page

### PR DESCRIPTION
**Pull request by nilshindermann**

### CODE REVIEW

File: app/(default)/page.tsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
+import Image from 'next/image';
 import Link from 'next/link';
 import { ReactElement } from 'react';
 
+import logo from './assets/nilshindermann_logo.svg';
```
Comment (nilshindermann) [inline app/(default)/page.tsx]: @llettieri is there a better way to import a svg logo?

This method works, but the build now fails and I don't want to just ignore it...

Comment (llettieri) [reply app/(default)/page.tsx]: In my projects I use a custom component for SVG images. Feel free to take a look in one of my projects.

File: app/(default)/page.tsx [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 import { ReactElement } from 'react';
 
+import logo from '../assets/nilshindermann_logo.svg';
+
 export default function Home(): ReactElement {
     return (
         <>
-            <h1 className="font-sans text-4xl font-bold text-primary md:text-6xl">
-                Nils Hindermann
+            <Image className="mb-8" src={logo} alt="nils hindermann" />
```
Comment (llettieri) [inline app/(default)/page.tsx:10]: Image looks great, but it seems a bit weird having your name in the title and then in the introduction.

Comment (llettieri) [reply app/(default)/page.tsx:10]: This is what I mean: 
image

**Review by llettieri — APPROVED** [review]

**Pull request merged**